update_licenses.yaml

```diff
@@ -0,0 +1,138 @@
+---
+remove_licenses:
+  - |-
+    // Copyright <value> The Chromium Authors. All rights reserved.
+    // Use of this source code is governed by a BSD-style license that can be
+    // found in the LICENSE file.
+  - |-
+    /**
+    Copyright <value> The Chromium Authors. All rights reserved.
+    Use of this source code is governed by a BSD-style license that can be
+    found in the LICENSE file.
+    **/
+  - |-
+    # Copyright <value> The Chromium Authors. All rights reserved.
+    # Use of this source code is governed by a BSD-style license that can be
+    # found in the LICENSE file.
+  - |-
+    <!--
+    Copyright <value> The Chromium Authors. All rights reserved.
+    Use of this source code is governed by a BSD-style license that can be
+    found in the LICENSE file.
+    -->
+  - |-
+    REM Copyright <value> The Chromium Authors. All rights reserved.
+    REM Use of this source code is governed by a BSD-style license that can be
+    REM found in the LICENSE file.
+add_licenses: 
+  - |-
+    // Copyright <value> The Flutter Authors
+    // Use of this source code is governed by a BSD-style license that can be
+    // found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
+  - |-
+    /**
+    Copyright <value> The Flutter Authors
+    Use of this source code is governed by a BSD-style license that can be
+    found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
+    **/
+  - |-
+    # Copyright <value> The Flutter Authors
+    # Use of this source code is governed by a BSD-style license that can be
+    # found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
+  - |-
+    <!--
+    Copyright <value> The Flutter Authors
+    Use of this source code is governed by a BSD-style license that can be
+    found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
+    -->
+  - |-
+    REM Copyright <value> The Flutter Authors
+    REM Use of this source code is governed by a BSD-style license that can be
+    REM found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
+update_paths:
+  include:
+    - /Users/mossman/Development/devtools
```

Comment:
This will be updated to a more generic path if/when we decide to merge this. Right now, using this to run locally and don't want to lose my changes.